server.js

```diff
@@ -31,7 +31,7 @@ app.use(session({
 
 // Connect to database
 mongoose.set('strictQuery', false)
-mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.connect(process.env.DATABASE_URL)
```

Comment:
use useNewUrlParser, useUnifiedTopology

Reply:
Added the useNewUrlParser and useUnifiedTopology